tests/checks/test_EmbeddingDistanceCheck.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+from shift_detector.checks.EmbeddingDistanceCheck import EmbeddingDistanceCheck
+from shift_detector.precalculations.Store import Store
+import pandas as pd
+
+
+class TestEmbeddingDistanceCheck(unittest.TestCase):
```

Comment:
I think you should also test a column that is not shifted and not only for a shift!
E.g. the same content (minimally changed) ;)